app/assets/stylesheets/layouts/starrylight.scss

```diff
@@ -1,86 +1,222 @@
-  b { color: #F3F3F3; }
+  a { color: inherit; }
+  a:hover div { background-color: $bg_color_menu_link_hover; }
+  div {
+    border-bottom: 1px solid $border_color_post_menu;
+    border-bottom: 1px solid rgba($border_color_post_menu, 0.5);
+  }
 }
-.post-expander { background-color: #D8D8D8; }
-
-#character-selector { font-weight: normal; }
-.post-icon, #current-icon-dropdown { background-color: #9498AC; }
-.post-author { background-color: #42416E; }
-.post-character, .post-screenname, .post-author { font-weight: normal; }
-.post-character, .post-screenname { background-color: #645F8A; }
-.post-screenname, .post-screenname a, .post-screenname a:visited { color: #C6C6E5; font-style: italic; }
-.post-character, .post-character a, .post-character a:visited, .post-author, .post-author a, .post-author a:visited { color: #F3F3F3; }
-
-/*.selected-icon { border-color: #409CFF; }*/
-
-.search-box { background-color: #645F8A; color: #FEFEFE; }
-.spacer-alt { background-color: #645F8A; color: #FEFEFE; }
-
-tr.gallery-tags .tag-item { background-color: #968BB4; color: #F7F7F7; }
-.tag-item { color: #444158; }
-a { color: #666284; }
-a:visited { color: #484357; }
-
-.flash.success, .flash.error, .flash.inbox { background-color: #CDC7E7; }
-.pagination .current { background-color: #9692A8; }
-.pagination a { color: #F3F3F3; }
-.pagination a:visited { color: #C6C6E5; }
-.pagination .disabled { color: #58546B; }
-.pagination a:hover { background-color: #9498AC; }
+
+$font_color_selectionpopup: #DADADA;
+$bg_color_selectionpopup: #6F658D; // selection popups like character/alias on posts#editor
+$bg_color_expander: #D8D8D8; // expander on posts
+
+.selector-popup { background-color: $bg_color_selectionpopup; color: $font_color_selectionpopup; }
+.post-expander { background-color: $bg_color_expander; }
+
+$bg_color_post_icon: #9498AC;
+$bg_color_icon_selector: #9692A8;
+$font_color_icon_selector: #DADADA;
+
+.post-icon { background-color: $bg_color_post_icon; }
+#current-icon-dropdown { background-color: $bg_color_post_icon; }
+#reply-icon-selector { background-color: $bg_color_icon_selector; color: $font_color_icon_selector; }
+
+$bg_color_post_character: #645F8A;
+$bg_color_post_screenname: #645F8A;
+$bg_color_post_author: #42416E;
+//$bg_color_post_character_narrow: $colors_lightbrown;
+//$bg_color_post_author_narrow: $colors_brown; // these are flipped from not-narrow mode to make the quick-switcher blend in
+// TODO: Should these flip on starries too?
+$font_color_post_info: #F3F3F3;
+
+.post-character { font-weight: normal; background-color: $bg_color_post_character; }
+.post-character, .post-character a, .post-character a:visited, { color: $font_color_post_info; }
+.post-screenname { font-weight: normal; background-color: $bg_color_post_screenname; }
+.post-screenname, .post-screenname a, .post-screenname a:visited { color: $font_color_link_visited; font-style: italic; } // TODO: clean this up when messages are fixed to not use it
```

Comment:
Can we split this color out into a `$font_color_post_screenname` and just set it to reference that color? Since, semantically, it's not a visited link.